mantidproject/mantid
# Estimate divergence

This creates a new algorithm to calculate the divergence of the instrument according to Windsor.

**To test:**

Try it out with various instruments. It is easiest to see the results via an instrument view of the output workspace.

*There  is no associate issue.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
